### ISSUE TITLE
Request: ability to hide information (config menu from taskbar button?)

### ISSUE THREAD

**Issue by ArchOwlen** [roguedex-dev/roguedex#15]
I probably won't use/need to know the ability, nature, and ivs of the pokemon I have on my team all the time, so the option to disable showing this information would be nice to free up more screen realistate 

Maybe some options could be moved into a taskbar button drop-down or settings menu?
This would also allow adjusting opacity of multiple pieces of the UI separately 
(Like how I mentioned the background, and weakness boxes as separate)

**Comment by ArchOwlen** [comment]
I made a branch to try getting the initial work done for this, but I somehow broke the entire plugin lol 

I'll take a look at it again later, but I can't figure out why it just breaks it entirely

**Comment by roguedex-dev** [comment]
Welcome to the hell that is extension debugging. What I usually do is place a debugger statement on top of e.g. create pokemon cards div function, then step over via debugger console until I hit the function that does not complete.
You can see which one it is because once you step over, you won't see return values e.g. for a function call or whatever, and instead it will proceed normally

**Comment by sirzento** [comment]
Yeah I would also like to be able to hide the ability, nature and IV's.
